docs/metadata-enumeration.md

```diff
@@ -26,4 +26,21 @@ Conditions: CAP_ODBC_METADATA_SUPPRESS_SQLCOLUMNS_API = false
 Conditions: CAP_ODBC_METADATA_SUPPRESS_PREPARED_QUERY = false OR CAP_ODBC_METADATA_SUPPRESS_EXECUTED_QUERY = false
 Example query: `SELECT * FROM default.testv1_batters`
 
-For all Tableau connector capabilities, please refer to capabilities [documentation](https://tableau.github.io/connector-plugin-sdk/docs/capabilities).
+## JDBC Metadata Enumeration
+There are six JDBC capabilities controlling JDBC metadata enumeration, They are all prefixed with `CAP_JDBC_METADATA_`. When metadata is enumerated by a SQL query, it will be logged with keyword `grpc-protocol-read-query-metadata`. When reading the metadata of columns from a table, the event will be logged with keyword `grpc-protocol-read-table-metadata`.
```

Comment:
In this section I also think its worth noting the underlying JDBC APIs we rely on by default:

JDBC API [DatabaseMetaData](https://docs.oracle.com/javase/7/docs/api/java/sql/DatabaseMetaData.html) getCatalogs, getSchemas, getTables, and getColumns. 

Without any capabilities set this is the default behavior.  In the JDBC case the capabilities below are far less likely to be used than in the ODBC case, where the defaults are very conservative.